docs/aws.md

```diff
@@ -44,16 +44,44 @@ brew install kubernetes-cli
 
 ## Setup your environment
 
-### Setting up a kops IAM user
+## Create an IAM user for kops
+In this example we will be using a dedicated IAM user to use with kops. This user will need basic API security credentials in order to use kops. Create the user and credentials using the AWS console. [More information](https://aws.amazon.com/documentation/iam/)
 
+The kops user will require the following IAM permission to function properly:
+```
+AmazonEC2FullAccess
+AmazonRoute53FullAccess
+AmazonS3FullAccess
+IAMFullAccess
+AmazonVPCFullAccess
+```
+
+If you already have a functioning AWS cli envrionment, you can create the kops user by doing the following:
```

Comment:
Can we move this block below `Testing your DNS setup` and above `Setting up a state store for your cluster?` The way it reads we are asking users to use the `aws` cli before we tell them how to install it :)